### PR TITLE
UI / Layout fixes, particularly for IE11

### PR DESCRIPTION
**Pull request by clhenrick**
small PR for https://github.com/mapd/mapd-immerse/issues/4594

### CODE REVIEW

**Comment by clhenrick** [comment]
Looks like this line of CSS is causing the horizontal scroll bar problem:

https://github.com/mapd/mapd3/blob/master/src/styles/common/_container.scss#L8

Given that the `config.useScrolling` defaults to `false`, I think it would make sense to have the `div.svg-wrapper`'s `overflow-x: hidden;` as the default setting, then somewhere in the chart change it if `useScrolling` is set to true.

**Comment by clhenrick** [comment]
@biovisualize let me know if you think this is the right place for this to happen: https://github.com/mapd/mapd3/pull/53/commits/53fe6ddb12bcd8d60b4753687016cfa73547877f

**Comment by clhenrick** [comment]
@biovisualize ready for your review when you have a minute

File: src/charts/helpers/auto-config.js [hunk 1]
```diff
@@ -27,6 +27,11 @@ export function autoConfigure (config, cache, data) {
   if (config.height === "auto" && cache.container) {
     newConfig.height = cache.container && cache.container.clientHeight || 0
   }
+  if (config.useScrolling && cache.svgWrapper) {
```
Comment (biovisualize) [inline src/charts/helpers/auto-config.js]: As you mentioned, I think autoconfig should just be for calculating any value to replace a configuration set to "auto" or to compute some values that are derived from the config. Maybe it should live right on chart.js build, like
```
cache.svgWrapper
      .style("flex", `0 0 ${config.chartWidth}px`)
      .style("height", `${config.height}px`)
      .style("overflow-x", config.useScrolling ? "scroll" : "hidden")
```

Comment (clhenrick) [reply src/charts/helpers/auto-config.js]: thanks, that makes more sense, will update.

**Comment by clhenrick** [comment]
@biovisualize ready for your re-review

**Pull request merged**